inst/scripts/import.R

```diff
@@ -0,0 +1,59 @@
+# Load mia R package
+library(mia)
+
+# Set file paths for input data
+biom_file_path <- "PATH_TO_BIOM_FILE"
+sample_meta_file_path <- "PATH_TO_SAMPLE_METADATA_FILE"
+tree_file_path <- "PATH_TO_PHYLOGENETIC_TREE_FILE"
+
+# Load data from the biom file into a SummarizedExperiment container
+x1 <- biomformat::read_biom(biom_file_path)
+se <- loadFromBiom(x1)
+
+# Convert the SummarizedExperiment to a TreeSummarizedExperiment container
+tse <- as(se, "TreeSummarizedExperiment")
+
+# We notice that the rowData fields do not have descriptibve names.
+# Hence, let us rename the columns in rowData
+names(rowData(tse)) <- c("Kingdom", "Phylum", "Class", "Order", 
+                         "Family", "Genus")
+
+
+# We also notice that the taxa names are of form "c__Bacteroidia" etc.
+# Goes through the whole DataFrame. Removes '.*[kpcofg]__' from strings, where [kpcofg] 
+# is any character from listed ones, and .* any character.
+rowdata_modified <- BiocParallel::bplapply(rowData(tse), 
+                                           FUN = stringr::str_remove, 
+                                           pattern = '.*[kpcofg]__')
```

Comment:
Hmmm, this kind of functionality might be actually added to `loadFromBiom`. Explore the function (`removeTaxaPrefix` or something, but in order to work, the function should first recognize colnames in rowData...)

What are the colnames in rowData? The function should recognize them because biom files are standardized. If they are not recognized, the function should be modified accordingly

Reply:
Yes that's a good idea.

Reply:
If I understand correctly, the colnames in rowData include :  "taxonomy1" "taxonomy2" "taxonomy3"....

Reply:
@TuomasBorman can comment on this, and how to do

Reply:
Hmmm, can you replace line 11 with

`se <- loadFromBiom(x1, removeTaxaPrefixes = TRUE, rankFromPrefix = TRUE)`

It should work so that you can remove lines 25-27 and 18-19 from your script

Reply:
ah, we were keeping BiocParallel::bplapply for future refence. but I'm sure we can find other ways to keep it. I will remove it.